.github/workflows/release.yml

```
@@ -0,0 +1,18 @@
+      - opened
+      - synchronize
+    branches:
+      - main
+
+jobs:
+  build:
+    uses: philipseo/actions/.github/workflows/pnpm-build.yml@main
+    with:
+      node-version: 20
+      pnpm-version: 9.11.0
+      npm-token: ${{ secrets.NPM_TOKEN }}
```

actionlint

context "secrets" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)